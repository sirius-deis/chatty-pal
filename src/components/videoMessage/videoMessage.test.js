import { render, screen } from "@testing-library/react";
import VideoMessage from "./videoMessage";

const message = {
  src: "video.mp4",
  message: "Watch this video",
  createdAt: "2023-10-01T12:00:00Z",
  isRead: false
}

describe("VideoMessage component", () => {
  it("should match snapshot", () => {
    const {container} = render(<VideoMessage message={message} />)
    expect(container).toMatchSnapshot();
  })
})
