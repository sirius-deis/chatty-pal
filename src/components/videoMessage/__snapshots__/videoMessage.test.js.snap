// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VideoMessage component should match snapshot 1`] = `
<div>
  <div
    class="sc-gEvEer feBbvx"
  >
    <video
      class="sc-eqUAAy kGDrMu"
      src="video.mp4"
    />
    <p
      class="sc-aXZVg bdiBRO"
    >
      Watch this video
    </p>
  </div>
</div>
`;
